.github/workflows/semgrep.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   analyze:
     name: Scan
     runs-on: ubuntu-latest
-    if: (github.actor != 'dependabot[bot]' || github.actor != 'renovate[bot]')
+    if: ${{github.actor != 'dependabot[bot]' && github.actor != 'renovate[bot]' && (!github.event.pull_request || github.event.pull_request.head.repo.full_name == github.repository)}}
```

Comment:
do we want this to just check if actor contains `[bot]`?

Reply:
Maybe, I was wondering if just the second check is enough and `github.event.pull_request.head.repo.full_name` is distinct for the bots